stock_putaway_product_form/hooks.py

```diff
@@ -1,21 +0,0 @@
-# Copyright 2020 Tecnativa - Sergio Teruel
-# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
-
-import logging
-
-from openupgradelib import openupgrade
-
-_logger = logging.getLogger(__name__)
-
-
-def post_init_hook(cr, registry):
```

Comment:
@victoralmau Why having removed this ? This is useful when you install the module on an existing environment and compute is bulky due to the number of products.

Reply:
I have removed because table not exists in V13 (https://github.com/OCA/OpenUpgrade/blob/13.0/addons/stock/migrations/13.0.1.1/openupgrade_analysis.txt#L128) and i thinked that this update is innecessary. 
I have removed too `openupgradelib` dependency from manifest related it.
Now exists `product_tmpl_id` field in `stock.putaway.rule` model by default.
Still necessary to keep on similar update (related to `stock.putaway.rule` table)?

Reply:
Indeed !

Reply:
OK, applied now.